src/index.html

```diff
@@ -7,11 +7,169 @@
+            max="100"
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input
+          type="date"
+          name="surname"
+          min="03-09-1922"
+          max="2022-09-03"
+          >
+        </label>
+        <label class="form__field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="surname"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group_indent">
+        <legend>Registration</legend>
+        <label class="form__field ">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            data-qa="field-placeholder"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group_indent">
+        <legend>An interesting fact about you!</legend>
+        <div class="item">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+            >
+              No
+          </label>
+        </div>
+        <label class="form__field">
+          What is your favorite color?
+          <input
+          autocomplete="on"
+          type="color"
+          name="head"
+          value="black"
+          >
+        </label>
+        <label class="form__field">
+          What time do you go to bed?
+          <input
+          autocomplete="on"
+          type="time"
+          name="appt-time"
+          step="1"
+          >
+        </label>
+        <label class="form__field">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          required
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option  disabled></option>
+          </select>
+        </label>
+        <label class="form__field">
+          How do you rate our work?
+          <input
+          autocomplete="on"
+          type="range"
+          name="rate"
+          min="0"
+          max="10"
+          step="2"
+          value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group_indent">
+        <legend>Additional info: </legend>
+        <label class="form__field">
+          Comments:
+          <textarea
+            name="feedback"
+            cols="20"
+            rows="2">
+          </textarea>
+        </label>
```

Comment:
```suggestion
          <textarea
            name="feedback"
            cols="20"
            rows="2"
         >



</textarea>
        </label>
```
давай виправимо цю поведінку:

https://user-images.githubusercontent.com/102413088/190143591-3c53205a-791a-400d-a97d-127e767cfb84.mov


[Відео Міши на цю тему](https://mate.academy/learn/css-advanced/forms#/video/885)